llvmlite/tests/test_ir.py

```diff
@@ -234,6 +235,34 @@ def test_metadata_null(self):
         self.assertInText("!{ i32* null }", str(mod))
         self.assert_valid_ir(mod)
 
+    def test_debug_info(self):
+        def order(d):
+            pairs = [(k, d[k]) for k in d]
+            pairs.sort()
+            return OrderedDict(pairs)
```

Comment:
So you can really spell this `OrderedDict(sorted(pairs.items()))`. It's more concise without losing readability :-)